services/housetables/src/main/resources/data.sql

```diff
@@ -0,0 +1,4 @@
+-- Initial value for feature toggle tables
+-- When enabling/disabling some feature, please ensure they are checked-in and reviewed through this file
+
+INSERT IGNORE INTO table_toggle_rule (feature, database_pattern, table_pattern, id, creation_time_ms) VALUES ('demo', 'demodb', 'demotable', DEFAULT, DEFAULT);
```

Comment:
What's the procedure for deleting rows/ disabling a feature entry?

Reply:
Didn't think thoroughly into it TBH. Some combination of manual operation in MySQL and merging a PR to remove a insert statement will be required.

Reply:
not blocking: can we check if `DELETE IF EXISTS` works? we have a singular way of adding non-urgent mysql changes?